common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -136,6 +136,16 @@
 
     private boolean forceRegister = false;
 
+    /**
+     *
+     * This configurable item defines interval of topics registration of broker to name server. Allowing values are
+     * between 10, 000 and 60, 000 milliseconds.
+     *
+     * If set to 0, newly created topics will be immediately reported to name servers and interval of periodical
+     * registration defaults to 10, 000 in milliseconds.
+     */
+    private int registerNameServerPeriod = 1000 * 30;
```

Comment:
registerNameServerPeriod is not a good name, maybe aliveICheckInterval is better. Another, do we need to polish here if no changes in a long time?

Reply:
origin default value